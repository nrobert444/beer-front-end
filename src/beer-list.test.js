import React from 'react';
import List from './beer-list.js';
import renderer from 'react-test-renderer';

it('list renders correctly', () => {
  const tree = renderer
    .create(<List />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});