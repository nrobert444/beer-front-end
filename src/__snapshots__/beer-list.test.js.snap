// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`list renders correctly 1`] = `
<div
  className="beer-list"
>
  <ul />
</div>
`;
